native/src/rapidsml_jni.cu

```diff
@@ -247,16 +281,74 @@ JNIEXPORT void JNICALL Java_com_nvidia_spark_ml_linalg_JniRAPIDSML_dgemm(JNIEnv*
     env->ThrowNew(jlexception, "Error freeing C from device");
   }
 
-  status = cublasDestroy(handle);
-  if (status != CUBLAS_STATUS_SUCCESS) {
-    env->ThrowNew(jlexception, "Error destroying cuBLAS handle");
-  }
-
   env->ReleaseDoubleArrayElements(A, host_A, JNI_ABORT);
   env->ReleaseDoubleArrayElements(B, host_B, JNI_ABORT);
   env->ReleaseDoubleArrayElements(C, host_C, 0);
 }
 
+JNIEXPORT void Java_com_nvidia_spark_ml_linalg_JniRAPIDSML_dgemmWithColumnViewPointer(JNIEnv *env, jclass, jint transa, jint transb, jint m, jint n,
+                                                                                      jint k, jdouble alpha, jlong A, jint lda, jdoubleArray B,
+                                                                                      jint ldb, jdouble beta, jlongArray C, jint ldc, jint deviceID)
+{
+  cudaSetDevice(deviceID);
+  raft::handle_t raft_handle;
+  cudaStream_t stream = raft_handle.get_stream();
+  jclass jlexception = env->FindClass("java/lang/Exception");
+
+  try {
+    cudf::column_view *A_cv_ptr = reinterpret_cast<cudf::column_view *>(A);
+    // init cuda stream view from rmm
+    auto c_stream = rmm::cuda_stream_view(reinterpret_cast<cudaStream_t>(stream));
+
+    auto size_B = env->GetArrayLength(B);
+    auto *host_B = env->GetDoubleArrayElements(B, nullptr);
```

Comment:
It seems I should use this RAII wrapper, I didn't find a . Can I just copy this `jni_utils.hpp` files directly to this project so that I can use them directly?